diff --git i/src/meson.build w/src/meson.build
index bced5b3..6d70757 100644
--- i/src/meson.build
+++ w/src/meson.build
@@ -6,7 +6,7 @@ libdex_opts = [
 ]
 
 libadwaita_dep       = dependency('libadwaita-1', version: '>= 1.7')
-libdex_dep           = dependency('libdex-1', version: '>= 1.0.0', default_options: libdex_opts)
+libdex_dep           = dependency('libdex-1', version: '>= 0.9.1', default_options: libdex_opts)
 flatpak_dep          = dependency('flatpak', version: '>= 1.9')
 appstream_dep        = dependency('appstream', version: '>= 1.0')
 xmlb_dep             = dependency('xmlb', version: '>= 0.3.4')
